### PR TITLE
chore: LemonButton cleanup

### PR DESCRIPTION
**Pull request by daibhin**
## Problem

Follow on from https://github.com/PostHog/posthog/pull/19540
Further to https://github.com/PostHog/posthog/pull/19416

## Changes

Merge two SCSS files into one. Removing any properties we no longer need

## How did you test this code?

Snapshots

### CODE REVIEW

File: frontend/src/scenes/insights/views/InsightsTable/InsightsTable.scss [hunk 1]
```diff
@@ -3,7 +3,6 @@
     align-items: center;
 
     .edit-icon {
-        font-size: 1rem;
```
Comment (benjackwhite) [inline frontend/src/scenes/insights/views/InsightsTable/InsightsTable.scss:6]: Other than this affecting the snapshots it all looks good (mostly trusting the snapshot system to have caught any issues)

File: frontend/src/scenes/insights/views/InsightsTable/columns/SeriesColumn.tsx [hunk 2]
```diff
@@ -28,7 +28,7 @@ export function SeriesColumnItem({
     const showCountedByTag = !!indexedResults.find(({ action }) => action?.math && action.math !== 'total')
 
     return (
-        <div className="series-name-wrapper-col">
+        <div className="series-name-wrapper-col space-x-1">
```
Comment (daibhin) [inline frontend/src/scenes/insights/views/InsightsTable/columns/SeriesColumn.tsx:31]: Only visual changes are now caused by this which adds a small gap so the button hover state does not touch the text

File: frontend/src/scenes/insights/views/InsightsTable/InsightsTable.scss (hunk 1 above)
Comment (daibhin) [reply frontend/src/scenes/insights/views/InsightsTable/InsightsTable.scss]: Yeah, worked through a lot of issues thanks to the snapshots. Fixed the icon size up so there's only (one) intentional change now

Comment (daibhin) [reply frontend/src/scenes/insights/views/InsightsTable/InsightsTable.scss]: Going to go ahead and merge this now that there is only a single snapshot change

**Pull request merged**